examples/stable-diffusion/text_to_image_generation.py

```diff
@@ -410,7 +432,11 @@ def main():
             control_image = Image.fromarray(image)
         kwargs_call["image"] = control_image
 
+    kwargs_call["quant_mode"] = args.quant_mode
+
     # Instantiate a Stable Diffusion pipeline class
+    import habana_frameworks.torch.core as htcore  # noqa: F401
```

Comment:
@imangohari1  `# noqa: F401` is there for a reason.  Make style would otherwise remove this thinking its not used.  But its crucial to include it so quantization works :)